static/docs/command-reference/import.md

```diff
@@ -139,22 +152,25 @@ deps:
       rev_lock: 0547f5883fb18e523e35578e2f0d19648c8f2d5c
 ```
 
-If the Git revision moves, such as a branch, this doesn't have much of an effect
-on the import/update workflow. However, for static refs such as tags (unless
-manually updated), or for SHA commits, `dvc update` will not have any effect on
-the import. In this cases, in order to actually "update" an import, it's
-necessary to **re-import the data** instead, by using `dvc import` again without
-or with a different `--rev`. For example:
+If the
+[Git revision](https://git-scm.com/book/en/v2/Git-Internals-Git-References)
+moves (e.g. branches), you may use `dvc update` to bring the data up to date.
```

Comment:
-> e.g. a/some branch (to make it play along with singular the Git revision and moves) 

Also, if you want to say it might be a branch you probably meant "a Git revision" above. Not sure I am right here, use your own sense of the language :)

Reply:
Agree, probably both singular and plural are valid here but singular sounds better. Updated.

> you probably meant "a Git revision" above.

I meant **the** specific revision used in the command example above (`--rev cats-dogs-v1`).

Reply:
But the for revision above is a tag, and you say it might be a branch in the same sentence.

Reply:
Correct. Still, it refers to **the** revision used in the `dvc import --rev ...` command, whatever it happens to be.

Reply:
If you change `the` to `a` then it won't be refering to anything above. That would be just some revision, this is what I meant from the start.

Reply:
I would then need to say "a revision used with the `--rev` option" which is much longer. Seems unnecessary since "the" is also correct.